src/tools/pub/cmd/pub-global.md

```diff
@@ -17,20 +17,20 @@ For example, say you want to run
 the Dart project generator, from the command line.
 
 ```terminal
-$ pub global activate stagehand
+$ dart pub global activate stagehand
 $ stagehand
 ```
 
 If this doesn't work, you might need to
 [set up your path](#running-a-script-from-your-path).
 
 To run a Dart script from within a package, or from a
-package that your package depends on, see [pub run](/tools/pub/cmd/pub-run).
+package that your package depends on, see [dart run](/tools/pub/cmd/pub-run).
```

Comment:
The URL needs to be updated. For now, let's use /tools/dart-tool.

Reply:
Ah ok @kwalrath 
can you please review it again?
Thanks